src/core/lwt_stream.mli

```diff
@@ -84,17 +84,17 @@ class type ['a] bounded_push = object
       is already blocked on [push], it raises {!Lwt_stream.Full}. *)
 
   method close : unit
-  (** Closes the stream. Any thread currently blocked on
-      {!Lwt_stream.bounded_push.push} fails with {!Lwt_stream.Closed}. *)
```

Comment:
I think maybe it should be `{!Lwt_stream.bounded_push#push}` but I find it hard to test or find resources for how to cross-reference methods.